datacommonsorg/website
# Merge redesign changes in to master

commit d5b325f0bc9103cc3ce413f0b47efa1e3c6ce000
Author: beets <beets@users.noreply.github.com>
Date:   Wed Jul 15 18:06:37 2020 -0700

    Redesign place (#76)

    Update style on place pages
    * Also fixed sidebar scroll issue
    * Layout updates for different window widths
    * Shrink "brand" link in header
    * Default to USA page instead of Mountain View
    * Fixed fixed-sidebar scroll sometimes overflowing into the footer. Pin it to the main chart container.

commit c7285635c0bb3036d6e35ef4226bf07367e285c2
Author: beets <beets@users.noreply.github.com>
Date:   Wed Jul 15 11:49:00 2020 -0700

    Homepage and header updates (#67)

    * Added COVID SA to Homepage's What's new
    * Update searchbox
    * Update header styles to match font size with tools pages

commit f32f9bf65f10025ce122b149cc7a52b8dce641db
Author: beets <beets@users.noreply.github.com>
Date:   Tue Jul 14 23:40:36 2020 -0700

    Apply new styles to the GNI page (#70)

    * Rename GNI to Timeline
    * Remove bulk download functionality
    * Share CSS with scatter
    * Fix up dev page

commit fe075b3eb6c50d26031198aebc19b31848512e24
Author: beets <beets@users.noreply.github.com>
Date:   Tue Jul 14 16:49:41 2020 -0700

    Merge master to the redesign branch (#66)

    * Add redirects for tool pages (#49)

    Moves
    gni -> /tools/timeline
    scatter -> /tools/scatter

    * Remove the oldest appengine version when auto push a new version to s… (#47)

    * Remove the oldest appengine version when auto push a new version to staging

    * Sort version by last deploy date and keep at lease 5 versions

    * Revert "Add redirects for tool pages (#49)" (#52)

    This reverts commit 8c6904b8d1793e7fa93e1f5c7b6b9244a664f50f.
    The change was meant as the first in a series of changes related to the redesign in a branch.

    * Display top 10 populated child places in place overview page; Fix various bugs in api and routing (#51)

    * Display child places in place overview page

    * Fix a bunch of crash and bugs

    * Do not return element when there is no child places

    * add "measuredProperty", add "placeTypes", change argstring (#50)

    * add measuredProperty, add PlaceTypes, change argstring of level 0 nodes to statvar dcid

    * remove duplicates at top level

    * add test for placeType

    * test

    Co-authored-by: Bo Xu <shifucun@users.noreply.github.com>

    * Remove US restriction for Place Auto Completion. (#55)

    * Remove US restriction for Place Auto Completion.

    * Fix search box hint text to remove US reference

    * Display GNI link on landing page. (#54)

    * Display GNI link on landing page.

    * open new tab for explore more link

    * Update style for explore more link

    * Add more docstrings and fix typos

    * Change StatVars in chart_config.json to new human readable ones. (#56)

    * Check in chart config for human readable stat vars.

    * Updated stat var IDs

    * Use new ChartStore cache for line chart api; Update to use human reada… (#58)

    * Use new CharStore cache for line chart api; Update to use human readable stats var everywhere.

    This removes mmethod, scaling factor and unit from the cache key.

    * Remove logging

    * Use typescript for place plage entry and react template (#59)

    * Move child place limit logic to routing level so sitemap can get the full list of child places (#63)

    * add initial commit to draw group line chart (#48)

    * add initial commit to draw group line chart

    * fix comments and refactor codes

    * address changes, add function to generate dashes and refactor codes

    * Update draw.ts

    * Follow ESlint to fix styles and add test case for dashes

    * improve function to generate dashes and address Carolyn's comments

    * fix comments and remove dup things

    * build the menu, update url upon click (#62)

    * build the gni menu using react js
    * add the function of updating the url once the statvar is selected

    * remove duplicated nodes in PV-tree (#65)

    * remove duplicated nodes in PV-tree

    * remove some comments

    Co-authored-by: Bo Xu <shifucun@users.noreply.github.com>
    Co-authored-by: qlj-lijuan <43854757+qlj-lijuan@users.noreply.github.com>
    Co-authored-by: Prashanth R <shanth@google.com>
    Co-authored-by: TJann <18621425+tjann@users.noreply.github.com>
    Co-authored-by: wh1210 <36181437+wh1210@users.noreply.github.com>

commit 4d7f0dd81c394cee0b288aa56c661233308e74a7
Author: beets <beets@users.noreply.github.com>
Date:   Tue Jul 14 13:33:09 2020 -0700

    Update styling on the scatter tool (#64)

    * Migrate from jquery 1.11 to the base version from base.html (3.5.1)
    * Remove jquery.ui
    * Cleaned up the plot selection form
    * Cleaned up dialog
    * Added welcome message
    * Remove "filter" button
    * Move form validation warnings from alerts to form hints
    * Slight code refactoring to support installing element handlers on load, rather on every chart drawing attempt
    * Fix layout due to displayed filtered place count
    * Address review comments, fix codacy warnings
    * fix eslint errors

commit 73d3b0f4ee069e4cd81ecda94780083d5513c87a
Author: beets <beets@users.noreply.github.com>
Date:   Sat Jul 11 11:30:30 2020 -0700

    Migrate most static pages to the new base template (#61)

    Migrate most static pages to the new base template
    * Also added /about
    * Moved some static templates to the main template directory
    * Some content/style cleanup
    * Moved static routes to a separate module
    * Fix tests
    * Fix Codacy warnings

commit 6f183c029f80e18858d68d28169d782f992bb079
Author: beets <beets@users.noreply.github.com>
Date:   Fri Jul 10 10:16:40 2020 -0700

    Homepage refresh (#60)

    Homepage refresh
    * Update structure of base.html
    * Update styling of header and footer
    * Add support for SASS and Bootstrap
    * Start stylesheet to be used for all static content pages

commit 5bb835b6f84f2968888f5956f351286e651a29b5
Author: beets <beets@users.noreply.github.com>
Date:   Thu Jul 9 16:54:50 2020 -0700

    /kg -> /browser (#57)

    * Move kg routes into a module
    * Convert dcid get param to path args
    * Add tests, update links sitewide
    * Add back dc.org homepage
    * Remove unused template

commit 7344e08d45d90bc04eaca56a06f7d24ecd6fd4e4
Author: beets <beets@users.noreply.github.com>
Date:   Wed Jul 8 09:03:22 2020 -0700

    Add redirects for tools (#53)

    /gni -> /tools/timeline
    /scatter -> /tools/scatter

    Added tests, and moved templates into a component.
